diff --git a/tests/test_currency.py b/tests/test_currency.py
--- a/tests/test_currency.py
+++ b/tests/test_currency.py
@@ -382,9 +382,4 @@ def suite():
             tearDown=doctest_teardown, encoding='utf-8',
             checker=doctest_checker,
             optionflags=doctest.REPORT_ONLY_FIRST_FAILURE))
-    suite.addTests(doctest.DocFileSuite(
-            'scenario_currency_rate_update.rst',
-            tearDown=doctest_teardown, encoding='utf-8',
-            checker=doctest_checker,
-            optionflags=doctest.REPORT_ONLY_FIRST_FAILURE))
     return suite
